backend/src/packages/chats/chat.controller.ts

```diff
@@ -98,7 +101,10 @@ class ChatController extends BaseController {
       method: 'GET',
       handler: (options) => {
         return this.findAll(
-          options as APIHandlerOptions<{ user: UserAuthResponseDto }>,
+          options as APIHandlerOptions<{
```

Comment:
Missing query validation

Reply:
@v0ldemar01 what do you mean by query validation. i get query params like an object with type:
{
query:string;
}

Reply:
It's type asserton

Reply:
I mean about validation on preHandler

Reply:
@v0ldemar01 so should i validate query value in controller? But in search field user can write any value.

Reply:
Yep, it should be only query field with string content

Reply:
done